MediaRecorder.bs

```diff
@@ -196,23 +216,44 @@ interface MediaRecorder : EventTarget {
     <li>If {{state}} is not {{inactive}}, throw an {{InvalidStateError}}
     {{DOMException}} and abort these steps.</li>
 
-    <li>If the {{MediaRecorder/stream}}'s <a>isolation properties</a> disallow access
+    <li>If the {{stream}}'s <a>isolation properties</a> disallow access
     from this {{MediaRecorder}}, throw a {{SecurityError}} {{DOMException}} and
     abort these steps.</li>
 
+    <li>If the {{stream}} is {{inactive}}, throw a {{NotSupportedError}}
+    {{DOMException}} and abort these steps.</li>
+
+    <li>Let |tracks| be the set of {{live}} tracks in the {{stream}}'s
+    [=track set=].</li>
+
+    <li>If the [=constrainedMimeType=] internal property specifies a media type,
+    container, or codec such that the User Agent cannot record a track in
+    |tracks| with it, throw a {{NotSupportedError}} {{DOMException}} and abort
+    these steps.</li>
+
+    <li>Set {{mimeType}} to the [=constrainedMimeType=] internal property,
+    extended by the media type, subtype and codecs parameter reflecting the
+    configuration used by the MediaRecorder to record all tracks in |tracks|.
+    The media type, subtype and any parameters in the [=constrainedMimeType=]
+    internal property, MUST NOT be changed. {{mimeType}} MAY be extended with
+    the profiles parameter [[RFC6381]] or further codec-specific parameters to
+    avoid ambiguity.</li>
```

Comment:
Instead of saying constrainedMimeType MUST NOT be changed, maybe split this into steps to make it clear? E.g.

 * Let |extendedMimeType| be the value of [=constrainedMimeType=]
 * Modify |extendedMimeType| by adding media type, subtype and codecs parameter reflecting the
    configuration used by the MediaRecorder to record all tracks in |tracks|, if not already present.
 * Set {{mimeType}} to the |extendedMimeType|